docs/advanced-features/event-driven-packaging.md

```diff
@@ -0,0 +1,177 @@
+<!-- markdownlint-disable -->
+# Event-Driven Packaging (EDP)
+
+> EDP is in private preview. Please contact us for details.
+
+## Overview
+
+Data tend to be created in logical batches by machines, people, and
+pipelines. Detecting these logical events from Amazon S3 events alone is
+complex and requires extensive custom logic.
+
+Quilt's *Event-Driven Packaging* service (EDP) smartly groups one
+or more Amazon S3 object events into a single event
+in [AWS EventBridge](https://aws.amazon.com/eventbridge/) so that
+you can easily trigger logical events like package creation that
+depend on batches rather than on individual files.
+
+Quilt EDP works well with services like [AWS Storage
+Gateway](https://aws.amazon.com/storagegateway/) and [AWS Data
```

Comment:
Just say works with any Amazon service that syncs data to S3 and any services that can be targeted from AWS EB (Lambda, batch, etc.)